src/test/java/com/github/dockerjava/core/command/StartContainerCmdImplTest.java

```diff
@@ -93,9 +93,15 @@ public void startContainerWithVolumes() throws DockerException {
 
         assertContainerHasVolumes(inspectContainerResponse, volume1, volume2);
 
-        assertThat(Arrays.asList(inspectContainerResponse.getVolumesRW()),
-                contains(new VolumeRW(volume1, AccessMode.ro), new VolumeRW(volume2)));
+        assertEquals(inspectContainerResponse.getMounts().size(), 2);
```

Comment:
you've changed order of expected and actual (error string will be wrong in case of fail)

to not repeat this mistake simply use `assertThat(actual, matcherForIt())` - thats more friendly to any reviewer or maintaineer